### DISCUSSION TITLE
심화 프로젝트 2주차 팀회고

### DISCUSSION THREAD

**Discussion by mayseunghwa** [codeit-bootcamp-frontend/1-Weekly-Mission#560] [회고]
# 왜 회고를 해야 하나요?
바쁘고 치열하게, 또 잘 버티고 있는 스프린트 1기 여러분!
정신없이 프로젝트를 하다 보면 현재 우리 팀의 방향성과 목표를 잊어버리기 쉬워요.
매주 팀원들과 일주일을 회고하며 프로젝트를 위한 원동력이 되었으면 좋겠어요.

# 회고에 대한 궁금증
회고와 일기의 차이는 무엇인가요?
- 일기는 일상에서 있었던 일들을 의식의 흐름대로 편하게 적는 것이라면,
회고는 함께 보고 듣고 경험한 사건과 현상들을 곱씹어 보고, 그중에 특히 되짚고 싶은 것들을 **'왜' 와 '어떻게'에 집중해서 기록**하는 것입니다.

# 회고 방법
<img width="636" alt="스크린샷 2024-01-17 오전 10 43 31" src="https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/assets/152246380/c1e8d800-0ef4-4f68-9b6c-076de80d7c3d">

회고는 **“KPT 회고”** 방식으로 진행해볼거예요. 아래 방법대로 회고해주시면 됩니다.

* 팀원들과 모여 그 주의 팀 프로젝트 활동을 돌아보며, '좋았던 점(Keep) / 아쉬웠던 점(Problem) / 시도해볼 것(Try)'으로 항목을 나누어 회고해주세요.
   - Keep : 현재 우리팀이 잘 하고 있어서 계속 유지하면 좋을 것 같은 부분
   - Problem : 현재 우리팀이 가지고 있는 문제
   - Try : Problem에 적은 것을 해결하기 위해 우리 팀이 다음에 시도해볼만한 것 (혹은 Keep 중에서도 더 개선하거나 시도해보고 싶은 것)
* [KPT 회고 피그마 템플릿](<https://www.figma.com/file/qjYHQFGLwPcG7eWfVN7dEE/%ED%9A%8C%EA%B3%A0-%ED%85%9C%ED%94%8C%EB%A6%BF_1%EA%B8%B0?type=whiteboard&node-id=0-1&t=jp6YMOCJvuXCw68E-0>)을 활용해서, 팀 회고를 진행하셔도 됩니다. 
   - 단 템플릿 사용시, 팀별로 복제해서 사용해주세요. 꼭 템플릿을 사용하지 않아도 되니, 팀별로 편한 방식으로 회고를 진행해주세요.
* **다만 이번주 토요일부터 프로젝트가 시작되기 때문에 2주차부터 진행할게요. 다음주 일요일까지, 팀 회고한 내용을 정리하여 댓글로 올려주세요.** 
(댓글에 예시 회고를 올려드릴테니, 참고해주세요. 피그마 템플릿으로 진행하셨다면 캡쳐본을 올려주셔도 돼요.)

## 참고글

- [KPT 회고란 무엇인가?](<https://brunch.co.kr/@jinha0802/35>)
- [팀문화의 탄생](<https://techblog.woowahan.com/2677/>)

**Comment by mayseunghwa** [comment]
# 🍀 0팀 (예시)

**Keep**
- 미팅 시간 약속을 모두 잘 지켜준다.
- 프로젝트 과정에서 기술 고민을 함께 하는 것이 서로의 학습에 도움이 된다.
- 새로운 제안을 했을 때, 모두 개방적으로 의견을 들어주어서 고맙다.
- 위워크에 모여서 하다보니, 프로젝트 진행이 빠르게 잘 이루어진다.

**Problem**
- 커밋 메시지가 명시적이지 않아서, 서로 어떤 작업을 했는지 파악하기 힘들었다.
- 급하게 코드를 작성하느라, 폴더 구조가 이상하다.
- 서로를 너무 배려하느라, 의견을 더 많이 나누지 못한다.

**Try**
- 협업을 위해, 커밋 메시지를 더 구체적으로 작성한다. (커밋 메시지 규칙 정하기)
- 폴더 구조를 다시 논의하여 폴더 구조 규칙을 만들고, 규칙에 맞추어 파일을 배치한다.
- 각자 하루에 한 번 이상 의견을 낸다.

**Comment by gw-lim** [comment]
# 3팀

### Keep

- 코어 타임을 정해 놓아서 서로 빠른 소통이 가능해 작업하기 편했다.
- 디자인이 빠르게 완성되고 공유가 되어서 좋았다.
- 빠르고 명확한 피드백 덕에 초반 기획을 탄탄히 할 수 있었다.
- 데일리 스크럼을 통해 각자의 상황을 공유하는 것이 효율적이었다.

### Problem

- 프론트 팀의 의견이 지배적인 것 같다. 백과 디자이너 분들의 의견을 더 많이 듣고 수용할 필요가 있다.
- 노션 페이지가 더 체계적으로 운영이 됐으면 좋을 것 같다. 칸반보드가 너무 길어짐.
- 회의가 너무 길어진다.

### Try

- 미리 회의 내용을 회의록에 정리해놓으면 좋을 것 같다.
- 회의 limit 시간을 정해두면 좋을 것 같다.
- 파트별 세부적인 데드라인을 정해놓으면 좋을 것 같다.
- 피그마의 follow 기능을 잘 활용하면 좋을 것 같다.
- 피그마 파일의 업데이트 사항을 잘 정리할 방법이 필요해보인다.

**Comment by Eugene-A-01** [comment]
# 🌊8팀

![1주차 회고](https://github.com/codeit-bootcamp-frontend/1-Weekly-Mission/assets/70089733/fd8d92d3-a2e3-4e68-a929-a257f8f62c27)

https://retro.tools/MVgHufXKLt0xDFU5AlaS

**Comment by minhyeokG0** [comment]
# 1팀

**Keep**

- 게더타운에서 모여있어서 빠르게 문제점을 해결 할 수 있다.
- 코어타임이 잘 지켜진다
- 어려운 부분의 기능 구현은 다같이 고민해서 빠르게 해결할 수 있다.
- 다들 말을 완곡하게 해주셔서 트러블이 생기지 않아서 좋았습니다 ❤️
- 자신의 할 일을 다했어도 아직 할 일을 완료하지 못한 팀원을 같이 쉐어링하면서 도와줘서 감사합니다

**Problem**

- 근거에 기반한 의견을 제시 하지 못하여 팀원분들을 설득하지 못하는 일이 발생하였다
- 협업이 처음이다보니 본인 의견을 명확하게 제시하지 못해서 오해하는 일이 생겼다.
- 깃헙을 잘 활용하고 있지 못한 것 같다.
- 시간 분배가 효율적으로 이루어지지 않는 것 같다.

**Try**

- 노션 캘린터 일정에 일정을 상세히 작성해서 효율적인 시간 관리를 하자! 🌈
- 아침 회의에서 작업을 분배한 뒤 현재 작업중인 부분을 게더타운 상태 메시지에 적어두자!
- 의견 및 답변을 근거에 기반해서 구체적으로 제시하자자
- 깃허브에서 활용할 수 있는건 다 활용해보자
- keep을 계 keep하자

**Comment by youdame** [comment]
# 9팀

<img width="812" alt="스크린샷 2024-01-29 오전 11 36 08" src="https://github.com/codeit-bootcamp-frontend/1-Weekly-Mission/assets/112458620/cab1bec5-ed38-4d60-88ea-3ab195b67c83">

**Comment by hyejungan** [comment]
https://lightning-eyelash-06c.notion.site/7-6aa0f8bf8d3a48f1882f5103dfcbda35?pvs=4
노션에 정리한 7팀 회고

**Comment by ayden94** [comment]
<img width="1824" alt="스크린샷 2024-01-30 오후 12 23 03" src="https://github.com/codeit-bootcamp-frontend/1-Weekly-Mission/assets/144667387/fc40cd3e-14f5-4ce4-a632-eb2622b2d965">

**Comment by Ssong-Q** [comment]
### 6팀

https://retro.tools/BRZUPLbobvxEcGcv86Kr

<img width="1280" alt="image" src="https://github.com/codeit-bootcamp-frontend/1-Weekly-Mission/assets/122101706/75f6de25-edb2-4969-8c94-2d81beb7100e">

<img width="1278" alt="image" src="https://github.com/codeit-bootcamp-frontend/1-Weekly-Mission/assets/122101706/fa4f2080-4f2a-46f7-8d3d-cf78d91bf9c1">

<img width="1280" alt="image" src="https://github.com/codeit-bootcamp-frontend/1-Weekly-Mission/assets/122101706/01ae1af5-35c9-4ca5-9ab1-8e7273edd5fd">

**Comment by hyeyoonS** [comment]
5팀 2주차 회고내용입니다~ 
https://www.notion.so/2-c786184372eb45ad9fbc43f1d01665e4?pvs=4
